pyanaconda/core/constants.py

```diff
@@ -390,3 +390,9 @@ class PayloadRequirementType(Enum):
 PARTITIONING_METHOD_MANUAL = "MANUAL"
 PARTITIONING_METHOD_INTERACTIVE = "INTERACTIVE"
 PARTITIONING_METHOD_BLIVET = "BLIVET"
+
+# Payload URL source types.
+URL_SOURCE_BASE = "BASE"
+URL_SOURCE_MIRRORLIST = "MIRRORLIST"
```

Comment:
Maybe the prefix of the constants could be `URL_TYPE_`?

Reply:
Fixed.